protos/sigstore_bundle.proto

```diff
@@ -43,7 +43,7 @@ message TimestampVerificationData {
         // transparency log, or in conjunction for a stronger trust model.
         // Clients MUST verify the hashed message in the message imprint
         // against the signature in the bundle.
-        repeated dev.sigstore.common.v1.RFC3161SignedTimestamp rfc3161_timestamps = 1;
+        repeated dev.sigstore.common.v1.RFC3161SignedTimestamp rfc3161_timestamps = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
The same reasoning is valid here as for the tlog entry, rfc3161 timestamps are optional.

Reply:
If somebody wants to omit rfc3161 timestamp, then they should completely omit `TimestampVerificationData`.
On the other hand, if they add `TimestampVerificationData`, then, I belive, they must provide at least one `rfc3161_timestamps` entry.

What is the purpose of adding `TimestampVerificationData` message with empty contents?

Reply:
Sorry, my bad. I read it as the `REQUIRED` was on the actual `TimestampVerificationData`. Ignore this comment :)